overrides/extension-override.json

```diff
@@ -3,6 +3,109 @@
         "autofill": {
             "state": "enabled"
         },
+        "cookie": {
+            "state": "enabled",
+            "settings": {
+                "trackerCookie": "disabled",
```

Comment:
Think we'll want this enabled on extension. Only on Chrome are we doing `nonTrackerCookie`.